app/uk/gov/hmrc/plasticpackagingtax/registration/models/registration/GroupDetail.scala

```diff
@@ -32,6 +32,12 @@ case class GroupDetail(
       TaskStatus.NotStarted
     else TaskStatus.Completed
 
+  def memberAlreadyPresent(newMember: GroupMember): Boolean =
+    this.members.exists(
+      groupMember => groupMember.customerIdentification1.equals(newMember.customerIdentification1)
```

Comment:
I thought you were doing duplicate checks against businessPartnerId?  Although this might work.  We have to have a cusId1 and it should be unique and different between the different group member types (post uk-company-only)

Reply:
I thought the company registration number would be unique so used it. Yeah the logic will change for different member types.

Reply:
It might be cleaner to implement `equals()` on `GroupMember`. This would nicely encapsulate the use of different checks for different org types.

Reply:
implemented equals